test/threadstest.c

```diff
@@ -435,9 +435,7 @@ static int _torture_rcu(void)
     writer2_done = 0;
     rcu_torture_result = 1;
 
-    rcu_lock = ossl_rcu_lock_new(1);
-    if (!rcu_lock)
-        goto out;
+    rcu_lock = ossl_rcu_lock_new(1, NULL);
```

Comment:
Why do we no longer test if `rcu_lock` is NULL?

Reply:
because I'm an idiot.  will fix